sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -184,24 +149,39 @@ object ScalaNativePluginInternal {
       val mainClass = selectMainClass.value.getOrElse(
         throw new MessageOnlyException("No main class detected.")
       )
-      val classpath = fullClasspath.value.map(_.data).filter(_.exists)
-      val entry     = nir.Global.Top(mainClass.toString + "$")
-      val cwd       = nativeWorkdir.value
-
-      tools.Config.empty
+      val classpath =
+        fullClasspath.value.map(_.data.toPath).filter(f => Files.exists(f))
+
+      val nativeLibJar =
```

Comment:
Given that subproject is called `nativelib`, I'd avoid capitalization for `Lib` here and everywhere else in this PR (when referring to `{native, scala, java, aux}lib`).